README.md

```diff
@@ -98,7 +129,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer...
+They tell Express how to extract data from forms.
```

Comment:
Yes, although the `.json` line does something different in addition to what you've already described.